src/main/java/com/bandwidth/iris/sdk/model/ImportTnCheckerPayload.java

```diff
@@ -8,9 +8,16 @@
 @XmlAccessorType(XmlAccessType.FIELD)
 public class ImportTnCheckerPayload extends BaseModel {
 
+    @XmlElement( name = "SiteId")
+    private int siteId;
+
+    @XmlElement( name = "SipPeer")
```

Comment:
```suggestion
    @XmlElement( name = "SipPeerId")
```
I think this should be `SipPeerId` as per the documentation